src/parse-options.js

```diff
@@ -4,6 +4,8 @@ const DEFAULT_OPTIONS = {
   nodeBin: 'node',
   maxAttempts: 3,
   protractorArgs: [],
+  // set logColor to one of the colors available in 'chalk' - https://github.com/chalk/chalk
```

Comment:
What about linking directly to the [ansi-styles color list](https://github.com/chalk/ansi-styles#colors)